internal/integration/emitter_test.go

```diff
@@ -83,7 +112,298 @@ func decodePromMetrics(src io.Reader) (*prometheus.MetricFamiliesByName, error)
 	return &mfs, nil
 }
 
-func TestNewTelemetryEmitterWithRoundTripper(t *testing.T) {
-	e := NewTelemetryEmitter("http://myapi.com", "myAPIkey", time.Second*20)
-	assert.Equal(t, "myAPIkey", e.apiKey)
```

Comment:
Why removing this test?

Reply:
because the key is not longer set in the `telemetryEmitter`.